post merge updates: block explorers (#7473)
ethereum/ethereum-org-website: src/content/developers/docs/data-and-analytics/block-explorers/index.md

## Data {#data}

> Ethereum is transparent by design so everything is verifiable. Block explorers provide an interface for getting this information. And this is for both the main Ethereum network and the testnets, should you need that data. Data is divided into execution data and consensus data. The execution data refers to the transactions that have been executed in a specific block. The consensus data refers to the blocks themselves and the validators who proposed them.

Reviewer: > Data is divided into execution data and consensus data. The execution data refers to the transactions that have been executed in a specific block. The consensus data refers to the blocks themselves and the validators who proposed them.

Isn't this something we're assuming people understand as a prerequisite?

Author: just reiterating to justify the layout of the rest of the page.

Here's a summary of the types of data you can get from a block explorer.

### Execution data {#execution-data}

> New blocks are added to Ethereum every 12 seconds (unless a block proposer misses its cue) so there's a near-constant stream of data that gets added to block explorers. Blocks contain a lot of important data that you may find useful:

Reviewer: ```suggestion
New blocks are added to Ethereum every 12 seconds (unless a block proposer misses its cue), so a near-constant stream of data gets added to block explorers. Blocks contain a lot of important data that you may find useful:
```

Would it still be accurate to say turn instead of cue here? Seems simpler for non-natives.

```suggestion
New blocks are added to Ethereum every 12 seconds (unless a block proposer misses its turn), so a near-constant stream of data gets added to block explorers. Blocks contain a lot of important data that you may find useful:
```

**Standard data**

- Block height – The block number and length of the blockchain (in blocks) on creation of the current block.

> - Timestamp – The time at which a miner mined the block.

Author: ```suggestion
- Timestamp – The time at which a block was proposed.
```

> - Timestamp – The time at which a miner mined the block.

Reviewer: This should be changed

- Transactions – The number of transactions included within the block.

> - Fee recipient – The address that received gas fee tips from transactions

Reviewer: ```suggestion
- Fee recipient – The address that received gas fee tips from transactions.
```

- Block Reward – The amount of ETH awarded to the validator who proposed the block.
- Size – The size of the data within the block (measured in bytes).
- Gas used – The total units of gas used by the transactions in the block.
- Gas limit – The total gas limits set by the transactions in the block.

> - Base fee per gas - minimum multiplier required for a transaction to be included in a block

Reviewer: Consistency.

```suggestion
- Base fee per gas - The minimum multiplier required for a transaction to be included in a block.
```

Reviewer: I don't love this explanation. Multiplier of what?

> - Burnt fees - how much ether is burned in the block

Reviewer: ```suggestion
- Burnt fees - How much ETH is burned in the block.
```

### Network {#network}

> Of course there's some data that speaks to the health of the network.

Reviewer: This is very odd phrasing.

> Of course there's some data that speaks to the health of the network.

Reviewer: I don't love this but it's better than what we have 😅 feel free to edit/expand/delete

```suggestion
Some block data is concerned about the health of Ethereum more holistically.
```

## Consensus layer data {#consensus-layer-data}

### Epoch {#epoch}

> Randomized committees of validators are created at the end of every epoch (every 6.4 minutes) for security reasons. Epoch data includes:

Reviewer: Original had weird sentence structure. What about:

```suggestion
For security reasons, randomized committees of validators are created at the end of every epoch (every 6.4 minutes). Epoch data includes:
```
